knowledge-base/customize-wpf-report-viewer.md

```diff
@@ -0,0 +1,143 @@
+---
+title: How to Customize the WPF Report Viewer
+description: Edit WPF Report Viewer template
+type: how-to
+page_title: Customize the WPF Report Viewer appearance
+slug: customize-wpf-report-viewer
+position: 
+tags: 
+ticketid: 1533694
+res_type: kb
+---
+
+## Environment
+<table>
+	<tbody>
+		<tr>
+			<td>Product</td>
+			<td>Progress® Telerik® Reporting</td>
+		</tr>
+		<tr>
+			<td>Report Viewer</td>
+			<td>WPF</td>
+		</tr>
+	</tbody>
+</table>
+
+
+## Description
+The article explains how to customize the appearance of the [WPF Report Viewer](../wpf-report-viewer-intro)
+
+## Suggested Workarounds
+Make sure you merge the resource dictionaries from the XAML files rather than from the default _Telerik.ReportViewer.Wpf.Themes_ assembly. See the article section [Setting Customizable Implicit Style for the WPF Report Viewer Example](../wpf-report-viewer-implicit-styles#setting-customizable-implicit-style-for-the-wpf-report-viewer-example) for details. The reason is that we need to modify the XAML files to add new controls and functionality to the viewer.
+
+Let's add two new buttons in the viewer's toolbar in _Telerik.ReportViewer.Wpf.xaml_ file of the corresponding theme:
```

Comment:
I searched up two times to see which one is the "corresponding theme" :)
Can we insert a sentence here stating that "In our demo we'll modify the report viewer template for Office 2019 theme and will change the file Telerik.ReportViewer.Wpf.xaml.

Reply:
Changed as suggested